packages/graph/hash_graph/lib/graph/src/store/postgres/knowledge/entity/mod.rs

```diff
@@ -427,6 +428,7 @@ impl<C: AsClient> PostgresStore<C> {
 
 #[async_trait]
 impl<C: AsClient> EntityStore for PostgresStore<C> {
+    #[tracing::instrument(level = "info", skip(self, properties))]
```

Comment:
Please do tell if you see an `instrument` attribute that doesn't skip potentially large/expensive payloads. These should be put into the request trace